### DISCUSSION TITLE
Some questions on Primeflex migration guide section in the v18 docs

### DISCUSSION THREAD

**Discussion by JIBIN-P** [primefaces/community#2708] [PrimeNG] [answered]
In the migration guide, It is mentioned
There are a couple of utility classes that are not migrated as they have no counterparts, use flexbox utilities instead as replacements.
formgrid
formgroup
formgroup-inline
col
col-fixed
field
field-checkbox
field-radiobutton
reset

I am using Primeflex for the layout of my forms and also some other things as well so this comes as a blow, It's nice that there's a utility available to convert most of the cases but from the above, I was relying on formgrid, col, field classes. Can anybody suggest what flexbox utilities can replace this or do I have to create a flexbox utility classes that replicate the same behavior and replace all these classes (painful)?

**Comment by qwadrox** [comment, marked as answer]
@JIBIN-P I would simply use [TailwindCSS](https://tailwindcss.com/docs/grid-template-columns) instead of those classes when possible. For example, instead of `formgrid` and `col` I would use `grid`: with `col-span-x` 
Instead of `formgroup-inline` I would use: `flex flex-row`.

That's what the docs say, basically: 

> "use flex box utilities instead as replacements"

To make quicker the migration, you can "find and replace" these classes with something from tailwind. 
Recently, I have migrated a project from primeflex to tailwindcss, and it was not that much problematic. I didn't use the migration tool.

**Reply by JIBIN-P** [reply]
Yepp, I think this is the way

**Reply by JIBIN-P** [reply]
Hii @qwadrox by any chance did you do a migration for https://primeflex.org/formlayout#advanced this kinda layout which was using the formgrid and grid, all my forms were using this and after the migration to tailwind everything seems broken and I haven't got around any easy help by flex do I have to restructure my whole form now or is there hoper by keeping the same structure and just change certain classes.
